README.md

```diff
@@ -26,6 +26,7 @@ jobs:
     steps:
     - name: Fetch Dependabot metadata
       id: dependabot-metadata
+      if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' }}
```

Comment:
As discussed in https://github.com/dependabot/fetch-metadata/issues/317

This check would be enough to prevent invalid users.

Reply:
This should definitely go in the README regardless of how we end up changing the code internals itself!